conf/igenomes.config

```diff
@@ -47,5 +47,24 @@ params {
       snpeffDb               = "GRCh38.86"
       vepCacheVersion        = "95"
     }
+    'GRCm38' {
+      acLoci                 = "${params.igenomes_base}/Mus_musculus/Annotation/ASCAT/fake.loci"
+      acLociGC               = "${params.igenomes_base}/Mus_musculus/Annotation/ASCAT/fake.loci.gc"
+      bwaIndex               = "${params.igenomes_base}/Mus_musculus/Sequence/BWAIndex/genome.fa.{amb,ann,bwt,pac,sa}"
+      chrDir                 = "${params.igenomes_base}/Mus_musculus/Sequence/Chromosomes"
+      chrLength              = "${params.igenomes_base}/Mus_musculus/Sequence/Length/Homo_sapiens_assembly38.len"
+      dbsnp                  = "${params.igenomes_base}/Mus_musculus/Annotation/mgp.v6.merged.norm.snp.indels.sfiltered.vcf.gz"
+      dbsnpIndex             = "${params.igenomes_base}/Mus_musculus/Annotation/mgp.v6.merged.norm.snp.indels.sfiltered.vcf.gz.tbi"
+      dict                   = "${params.igenomes_base}/Mus_musculus/Sequence/WholeGenomeFasta/genome.dict"
+      fasta                  = "${params.igenomes_base}/Mus_musculus/Sequence/WholeGenomeFasta/genome.fa"
+      fastaFai               = "${params.igenomes_base}/Mus_musculus/Sequence/WholeGenomeFasta/genome.fai"
+      germlineResource       = "${params.igenomes_base}/Mus_musculus/Annotation/GermlineResource/gnomAD.r2.1.1.GRCh38.PASS.AC.AF.only.vcf.gz"
```

Comment:
We have more info on that file here: https://github.com/nf-core/sarek/blob/dev/docs/usage.md#--germlineresource